examples/oauth2/docker-compose.yml

```diff
@@ -13,18 +13,12 @@ services:
     image: redis
   oauth:
     build: ./auth-server
-    image: auth-server
     ports:
       - "3000:3000"
-    volumes:
-      - ./auth-server:/opt/app/auth-server
   client:
     build: ./client
-    image: client
     links:
       - gateway
       - oauth
     ports:
-      - "3001:3001"
-    volumes: 
-      - ./client:/opt/app/client
+      - "3001:3000"
```

Comment:
OK, I see. I was confused by [this line](https://github.com/3scale/apicast/pull/153/files#diff-dbc444dfe31dafb8c138ea1773769119R7) I guess it's not needed?